.travis.yml

```diff
@@ -30,10 +30,8 @@ before_install:
 
 # Install packages
 install:
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION --file requirements.txt
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION jupyter
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION pyflakes
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION pep8
+  - conda install --yes python=$TRAVIS_PYTHON_VERSION scipy matplotlib h5py jupyter
```

Comment:
aktually the way it was written was good:
- install basic dependencies via `requirements.txt`
- install optional dependency `jupyter` for gui tests in travis
- install checkers `pyflakes` and `pep8`

if you don't had something else in mind, pls revert the changes in this file